### ISSUE TITLE
Some questions about CHIEF pre-training

### ISSUE THREAD

**Issue by KKIverson** [hms-dbmi/CHIEF#43]
Congratulations on this work on CHIEF.
I have a few questions about pre-training. I noticed that the supplementary file mentioned that 3 losses were used when pre-training the CHIEF model. Formula (3) is used to calculate the bag-level classification loss, and formula (4) is used to calculate the instance-level top-k loss. I understand both 3 and 4. But I don’t understand the loss in formula (5).
First, there are two negative signs in formula 5. Is there a typo (should there be only 1 negative sign) ?  Second, for each bag(G), whether it is negative or positive, does the numerator calculate the similarity between each instance in G and the instance in the positive bag? If bag(G) is a negative sample, will it be counterproductive? Because your goal is to maximize the intra-class similarity and minimize the inter-class similarity. 
Thank you!
![企业微信截图_17338104562370](https://github.com/user-attachments/assets/43f79469-d049-4cad-bdd4-31599aa0260c)
![企业微信截图_17338104499190](https://github.com/user-attachments/assets/8a15bf63-0c8c-46d1-8074-241655369e81)
![企业微信截图_17338102479915](https://github.com/user-attachments/assets/40f37efe-5d26-4908-a531-b55f0286b62a)

**Comment by KKIverson** [comment]
Another small question: How big is this memory bank? Does it contain all the samples in this tissue source site? How is it maintained? Is it updated in each iteration?
